variables.tf

```diff
@@ -9,3 +13,257 @@ variable "tags" {
   description = "User-Defined tags"
   type        = map(string)
 }
+
+########################################
+# Bitbucket Vars
+########################################
+
+variable "admin_email" {
+  description = "email address for administrator"
+  type        = string
+}
+
+variable "admin_password" {
+  description = "initial password to use for administrator"
+  type        = string
+}
+
+variable "data_volume_size" {
+  description = "Size in GB of the data volume"
+  type        = number
+}
```

Comment:
Is there no reasonable default you can think of for the data volume?

Reply:
I made an assumption for the root volume but I figure for the data volume, that really ought to be given some thought.